src/forms/labels/labeled-field.js

```diff
@@ -83,17 +83,18 @@ function LabeledField ({
   input: { name },
   meta: { error, touched, invalid },
   className,
-  errorComponent: ErrorComponent = InputError,
+  errorComponent: ErrorComponent,
   labelComponent: LabelComponent = InputLabel,
   children,
   hideErrorLabel,
   ...rest
 }) {
+  const baseErrorProps = { error, invalid, touched, name }
```

Comment:
Maybe just pass these to the component directly rather than assigning it to a variable since `...rest` includes them anyway.

Reply:
@dpikt Since these values are destructured from props, aren't they not included in `...rest`?

Reply:
You're right, I got confused by the nesting. In that case we might want to pull them out a different way in case the custom component wants access to the other values stored in `input` and `meta`.